src/manipulate.jl

```diff
@@ -35,8 +43,11 @@ macro manipulate(expr)
         bindings = [expr.args[1]]
     end
     syms = symbols(bindings)
-    Expr(:let, Expr(:block,
-                    display_widgets(syms)...,
-                    esc(map_block(block, syms))),
-         map(make_widget, bindings)...)
+    Expr(:let,
+            Expr(:tuple,
+                map(sym->esc(sym), syms)...,
+                esc(Expr(:...,map_block(block, syms)))
+            ),
+            map(make_widget, bindings)...
+    )
```

Comment:
Yeah it is a bit hacky 😄 
> Is it possible now to display signals in an hbox or vbox?
Not right now, but haven't looked into what is required, haven't really looked into what Interact does on the js side either.